src/irr/asset/CGraphicsPipelineLoaderMTL.cpp

```diff
@@ -231,50 +256,64 @@ layout (set = 3, binding = 5) uniform sampler2D map_d;
 layout (set = 3, binding = 6) uniform sampler2D map_bump;
 #endif //_IRR_FRAG_SET3_BINDINGS_DEFINED_
 
-#include <irr/builtin/glsl/brdf/specular/fresnel/fresnel.glsl>
+#include <irr/builtin/glsl/bsdf/brdf/specular/fresnel/fresnel.glsl>
 
-#ifndef _IRR_FRAG_MAIN_DEFINED_
-#define _IRR_FRAG_MAIN_DEFINED_
+#ifndef _IRR_COMPUTE_LIGHTING_DEFINED_
+#define _IRR_COMPUTE_LIGHTING_DEFINED_
 
-#define Ia 0.1
-void main()
+irr_glsl_BSDFIsotropicParams irr_glsl_computeLighting()
```

Comment:
in the per-shader built-ins (which are not in built-in GLSL headers) you gon't need to prefix the names with `irr_glsl_`, just `irr_` will do